src/test/scala/com/cognite/sdk/scala/v1/ClientTest.scala

```diff
@@ -309,9 +315,19 @@ class ClientTest extends SdkTestSpec with OptionValues {
         badRequest,
         assetsResponse
       )
-    val client = retryingClient(badRequestBackendStub)
-    client.login.status().project shouldBe loginStatus.project
-    client.assets.list().compile.toList.length should be > 0
+    val client = new GenericClient[IO]("scala-sdk-test",
+      projectName,
+      "https://www.cognite.com/nowhere-at-all",
+      ApiKeyAuth("irrelevant", Some("randomproject"))
+    )(
+      implicitly,
+      new RetryingBackend[IO, Any](
+        badRequestBackendStub,
+        initialRetryDelay = 1.millis,
+        maxRetryDelay = 2.millis)
+    )
+    client.login.status().unsafeRunSync().project shouldBe loginStatus.project
+    client.assets.list().compile.toList.unsafeRunSync().length should be > 0
```

Comment:
this does not seem right

Reply:
Care to elaborate? :) It's supposed to do the same thing as the old test, but using IO + unsafeRunSync instead of the Id monad.